tester_app/client/webpack.client.rails.config.js

```diff
@@ -0,0 +1,45 @@
+// Run like this:
+// cd client && npm run build:dev
+// Note that Foreman (Procfile.dev) has also been configured to take care of this.
+
+// NOTE: All style sheets handled by the asset pipeline in rails
+
+const webpack = require('webpack');
+const config = require('./webpack.client.base.config');
+
+const devBuild = process.env.NODE_ENV !== 'production';
```

Comment:
@alexfedoseev: Are you suggesting we should have all those available?

Perfect example from @petehunt: https://github.com/petehunt/webpack-howto#6-feature-flags

@robwise if easy, good thing to drop in!